screens/aqi_screen/IndexPage.tsx

```diff
@@ -1,21 +1,130 @@
-import { StyleSheet, Text, View} from 'react-native'
-import React from 'react'
+import { StyleSheet, Text, View, Image, Pressable } from "react-native";
+import React, { FC } from "react";
+import { Ionicons } from "@expo/vector-icons";
+import { AntDesign } from '@expo/vector-icons';
+import { StackParamList } from "../../App";
+import { NativeStackScreenProps } from "@react-navigation/native-stack";
+type IndexPageScreenProps = NativeStackScreenProps<StackParamList, "IndexPage">;
 
-const IndexPage = () => {
+const IndexPage: FC<IndexPageScreenProps> = (props) => {
   return (
     <View style={styles.container}>
-    
-    <Text>Index Page</Text>
+      <Image
+        style={styles.bgImg}
+        source={require("../../assets/cloudSky3.jpg")}
+      />
+      <View style={{ flexDirection: "row", justifyContent:'space-between' }}>
+        <Pressable
+          style={styles.backIcon}
+          onPress={() => props.navigation.push("Main")}
+        >
+          <Ionicons name="chevron-back-outline" size={48} color="black" />
+        </Pressable>
+        <Pressable style={styles.closeIcon}
+        onPress={() => props.navigation.push("Main")} 
+        
+        >
+           <AntDesign  name="close" size={46} color="black" />
+        </Pressable>
+       
+      </View>
+
+      <View style={styles.wrapper}>
+        <Text style={styles.title}>Air Quality Index</Text>
+        <View style={[styles.colorBackground,styles.cardShadow,{ backgroundColor: '#19740d'}]}  
+        >
+          <Text style={styles.indexText}>0-50</Text>
+          <Text style={styles.indexText}>Good</Text>
+        </View>
+        <View style={[styles.colorBackground, styles.cardShadow, { backgroundColor: '#ffd60a'}]}>
+          <Text style={styles.indexText}>51-100</Text>
+          <Text style={styles.indexText}>Moderate</Text>
+        </View>
+        <View style={[styles.colorBackground,styles.cardShadow,{ backgroundColor: '#e7771b'}]}>
+          <Text style={styles.indexText}>101-150</Text>
+          <Text style={styles.indexTxt}>Unhealthy for Sensitive Groups</Text>
+        </View>
+        <View style={[styles.colorBackground,styles.cardShadow,{ backgroundColor: '#eb0a24'}]}>
+          <Text style={styles.indexText}>151-200</Text>
+          <Text style={styles.indexText}>Unhealthy</Text>
+        </View>
+        <View style={[styles.colorBackground, styles.cardShadow,{ backgroundColor: '#6a0674'}]}>
+          <Text style={styles.indexText}>201-300</Text>
+          <Text style={styles.indexText}>Very Unhealthy</Text>
+        </View>
+        <View style={[styles.colorBackground, styles.cardShadow,{ backgroundColor: '#5f0f40'}]}>
```

Comment:
You may want to add these colors to a constants file so you don't have to retype them, and they'll have a descriptive name.  Example:

```
export const colors = {
  green: "#19740d",
};
```

Then import it and use it like `backgroundColor:  colors.green`.  Just something that might help readability and maintainability!

Reply:
Thank you for your feedback. I implemented your suggestion